src/main/scala/atom/config-schema/SettingsGroup.scala

```diff
@@ -3,16 +3,15 @@ package laughedelic.atom.config
 import scala.scalajs.js
 
 class SettingsGroup[S <: ConfigSchema](
+  schema: S,
   val title: js.UndefOr[String] = js.undefined,
-  val description: js.UndefOr[String] = js.undefined,
-  val order: js.UndefOr[Int] = js.undefined,
   val collapsed: js.UndefOr[Boolean] = js.undefined,
-  schema: S,
```

Comment:
I moved this up as a first arg, because it's the only non-optional argument. I also wanted to put it in a separate parameter list, i.e. `new SettingsGroup(someSchema)(title = "foo", collapsed = true)`, but this doesn't work: https://github.com/scala-js/scala-js/issues/3285